### PR TITLE
feat: add labels to pod log navigation buttons

### PR DESCRIPTION
**Pull request by rbreeze**
A small/simple improvement. Will improve other buttons at a later date

CleanShot 2022-10-10 at 14 15 11@2x

Signed-off-by: Remington Breeze <remington@breeze.software>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

**Comment by alexec** [comment]
I think this is better. But I’d still rather all the logs in a scrollable window. 

Really, I want to find the errors.

File: ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.tsx [hunk 1]
```diff
@@ -505,22 +505,45 @@ interface PageInfo {
...
+            <NavButton disabled={!(info && info.canPageBack)} onClick={actions.left || (() => null)}>
+                Prev <i className='fa fa-angle-left' />
+            </NavButton>
+            <NavButton onClick={actions.bottom || (() => null)}>
+                Bottom <i className='fa fa-angle-down' />
+            </NavButton>
+            <NavButton onClick={actions.top || (() => null)}>
+                Top <i className='fa fa-angle-up' />
+            </NavButton>
             <div style={{marginLeft: 'auto', marginRight: 'auto'}}>
```
Comment (keithchong) [inline ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.tsx:534]:
```suggestion
            <div className={`page-info`}>
```

File: ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.scss [hunk 2]
```diff
@@ -65,13 +67,11 @@
         display: flex;
         margin-bottom: 1em;
         padding-left: 10px;
-        i {
+        .nav-container {
+            display: flex;
+            padding: 3px 8px;
```
Comment (keithchong) [inline ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.scss:72]:
```suggestion
        .page-info {
            margin-left: auto;
            margin-right: auto;
            padding: 3px 8px;
            text-overflow: clip;
            white-space: nowrap;
        }
        .nav-container {
            display: flex;
            align-self: flex-start; 
            padding: 3px 8px;
```

Comment (keithchong) [reply ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.scss:72]: Try this...replace lines 70-72, `.nav-container {` to `padding: 3px 8px;` with the above.

Comment (keithchong) [reply ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.scss:72]: Hey @rbreeze , your changes are fine.  I was just testing it out to see how the buttons look when you shrink the width of the browser.  This is what happens with and without my suggestion.  I guess it's just a matter of personal preference?  Feel free to disagree but I'll give it a LGTM.

With suggestion:
WithSuggestion

As is:
AsIs-WithoutSuggestion

Comment (rbreeze) [reply ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.scss:72]: Thanks for the review @keithchong! I agree, I like how it looks with your suggestion better.

**Review by alexec — APPROVED** [review]

**Pull request merged**